cli/bazelisk/bazelisk.go

```diff
@@ -13,7 +15,7 @@ import (
 	"github.com/bazelbuild/bazelisk/core"
 	"github.com/bazelbuild/bazelisk/repositories"
 	"github.com/buildbuddy-io/buildbuddy/cli/arg"
-	"github.com/buildbuddy-io/buildbuddy/cli/log"
+	bbLog "github.com/buildbuddy-io/buildbuddy/cli/log"
```

Comment:
nit: we want to prioritize using our custom logging package so it would be better IMO to alias the stdlib package instead. e.g.

```
golog "log"
```

Reply:
done